code/src/cljs/sixsq/nuvla/ui/main/views.cljs

```diff
@@ -30,9 +30,10 @@
             [sixsq.nuvla.ui.main.views-sidebar :as sidebar]
             [sixsq.nuvla.ui.messages.views :as messages]
             [sixsq.nuvla.ui.notifications.views]
-            [sixsq.nuvla.ui.panel :as panel]
             [sixsq.nuvla.ui.profile.subs :as profile-subs]
             [sixsq.nuvla.ui.profile.views]
```

Comment:
I think you can delete all requires that import views since they are now imported explicitly by the routes ns

Reply:
Nice!